packages/fxa-settings/src/components/Profile/index.tsx

```diff
@@ -22,7 +22,13 @@ export const Profile = () => {
       </h2>
 
       <div className="bg-white tablet:rounded-xl shadow">
-        <UnitRowWithAvatar />
+        <UnitRow
```

Comment:
It looks like we're going to want to wrap this in a `<Localized>` component, similar to how we're wrapping other `<UnitRows>` in `Profile/index.tsx`. We don't want to lose the l10n ID `avatar-heading` which will need to be copied from the `UnitRowWithAvatar` FTL file, and I'd double check that "Change" (`avatar-change-link`) and "Add" (`avatar-add-link`) look like they'll be covered with the `UnitRow` FTL file.

Reply:
I tried wrapping the `UnitRow` with a `<Localized id="avatar-heading" attrs={{ header: true }}>{...}</Localized>` but it adds a `Picture` text below the avatar.
![image](https://user-images.githubusercontent.com/40818234/113818373-4e07fe00-9795-11eb-8643-07283bad92ef.png)
Should I localize this instead, since that's where I assume we'd want it:
https://github.com/mozilla/fxa/blob/21e80ce3192ddbd2b9ce6acf8ada9b5ea34f6bbe/packages/fxa-settings/src/components/UnitRow/index.tsx#L133-L135

Reply:
I see what you mean. It looks like the localization header renders as as React `children` when done that way.

I took a look at the other unit rows and noticed we have the localization in `Profile/en-US.ftl` and specify `.header` in there. I tried [this out](https://gist.github.com/LZoog/86803fe44a4b451259a3d2a766e16dcc) and it looks like it's rendering correctly that way. Try that out and let me know if it's looking good to you too!

Reply:
Yes, this works, thank you!